src/suggestions.js

```diff
@@ -59,6 +61,9 @@ export default class SimpleteSuggestions extends HTMLElement {
 			let selector = container && container.getAttribute(attr);
 			this[prop] = selector || DEFAULTS[prop];
 		});
+
+		// remove focussable elements
+		find(this, FOCUSSABLE_ELEMENTS).forEach(el => el.setAttribute("tabindex", "-1"));
```

Comment:
Minor readability adjustment (IOW line length 🙃 ):

```suggestion
		find(this, FOCUSSABLE_ELEMENTS).
			forEach(el => el.setAttribute("tabindex", "-1"));
```

I also opted to remove the comment here, as it seemed to redundantly describe the code; YMMV 🤷